test/controllers/DocumentationControllerSpec.scala

```diff
@@ -16,41 +16,28 @@
 
 package controllers
 
-import java.io.FileNotFoundException
-
 import akka.stream.Materializer
-import org.scalatest.mockito.MockitoSugar
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import play.api.http.Status._
-import play.api.test.FakeRequest
-import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
-
-class DocumentationControllerSpec extends UnitSpec with MockitoSugar with WithFakeApplication {
+import play.api.mvc.Result
+import play.api.test.Helpers.{defaultAwaitTimeout, status, contentAsString}
+import play.api.test.{FakeRequest, Injecting}
+import utils.BaseSpec
 
-  private implicit val materializer: Materializer = fakeApplication.materializer
+import scala.concurrent.Future
 
-  private lazy val controller = fakeApplication.injector.instanceOf[DocumentationController]
-  private lazy val applicationRamlContent = getResourceFileContent("/public/api/conf/1.0/application.raml")
+class DocumentationControllerSpec extends BaseSpec with GuiceOneAppPerSuite with Injecting {
```

Comment:
Does the Scala Formatting only take action on the "RealTimeIncomeInformationControllerSpec"?

Reply:
In this PR we have only ran the formatter in select files, we will run the formatter against the whole repo in a future and separate PR